Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -138,7 +138,7 @@ void UGlobalStateManager::GetSingletonActorAndChannel(FString ClassName, AActor*
 	OutActor = nullptr;
 	OutChannel = nullptr;
 
-	UClass* SingletonActorClass = FindObject<UClass>(ANY_PACKAGE, *ClassName);
+	UClass* SingletonActorClass = LoadObject<UClass>(nullptr, *ClassName);
```

Comment:
What triggered this change? did we use to load these on load before? Even though the impact of loading a UClass should be cheap and only happen the first time an object is requested it would be good to avoid LoadObject during runtime if possible.

Reply:
This was changed due to @m-samiec's comment saying that if a singleton is a blueprint class, and we use `FindObject` it might not be loaded.

...though since the game is running I don't know how that would be possible.

Reply:
Blueprints are loaded via references even at runtime. So if there's a blueprint in a different level, or a non-loaded sublevel, it won't be loaded.
There's no performance implication either, as LoadObject does a FindObject first.